index.html

```diff
@@ -20,27 +20,27 @@
     </nav>
     <div class="headline">
     <div class="description">
-      <h1 class="name"> I am Aime
-          Glad to see you!</h1>
+      <h2 class="name"> I am Aime
+          Glad to see you!</h2>
          
       <p class="text"> I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
            </p>
       </div>
       <div class="contact">
         <button>
 
-          <h2 class="media">LET'S CONNECT:</h2>
+          <h2>LET'S CONNECT</h2>
         
         </button>
         <ul>
```

Comment:
[OPTIONAL] As it was missed by the previous reviewer, I am adding it as Optional, however thay are the core requirement for this project so I higly recommend you take them into consideration
- Please make sure the The container of the toolbar section is a `header` tag.
* Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.The container of the section is a header tag.
example: 
```
<header>
 <a> logo </a> 
 <nav> hamburger icon </nav> 
</header>
```
- Please make sure The container of the  Headline section is a `section` tag.
* It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.The container of the section is a section tag.
example:
```
<section class='headline'>
  <h1> I am ... glad to see you </h1>
  <p> "I'm a software ..." </p>
    ...
</section>
```
_**This way will be semantically correct and the screen readers and other technologies will thank you**_ :pray: